Description: Debian specific tweaks for NetworkManager systemd service file
 - Do not install NetworkManager into the network.target, as this will lead to a
   dependency loop (NetworkManager is installed in /usr).
Author: Michael Biebl <biebl@debian.org>
--- a/data/NetworkManager.service.in
+++ b/data/NetworkManager.service.in
@@ -1,7 +1,5 @@
 [Unit]
 Description=Network Manager
-Wants=network.target
-Before=network.target
 
 [Service]
 Type=dbus
